candump.c

```diff
@@ -383,22 +388,37 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
-	for (i=0; i < currmax; i++) {
+	memset(&event_setup, 0, sizeof(event_setup));
+	event_setup.events = EPOLLIN; /* prepare the common part */
+	fd_epoll = epoll_create1(0);
+	if (fd_epoll < 0) {
+		perror("epoll_create1");
+		return 1;
+	}
 
+	for (i=0; i < currmax; i++) {
```

Comment:
please add a space around the `=`

Reply:
Not my line of code, it just jot shuffled around and appeared in the diff. Changed anyhow. (Most for() loops in the existing code don't have spaces around `=`.)

Reply:
Thanks, the original code wasn't following the Linux kernel coding style. But if you touch the code, we can convert it :smile: